Uplift/Controllers/SportsDetailViewController.swift

```diff
@@ -9,23 +9,156 @@
 import UIKit
 
 class SportsDetailViewController: UIViewController {
+     
+    private let collectionView = UICollectionView(frame: .zero, collectionViewLayout: StretchyHeaderLayout())
     
-    private var post: Post
+    private var post: Post!
+    private var section: Section!
```

Comment:
can we drop the `!` and just set both of these values before we call `super.init`? that way, we'll get a compilation error if someone happens to remove the initialization